brave/src/main/java/com/linecorp/armeria/client/brave/ClientRequestContextAdapter.java

```diff
@@ -44,9 +44,9 @@
     @SuppressWarnings("ClassNameSameAsAncestorName")
     private static final class HttpClientRequest extends brave.http.HttpClientRequest {
         private final ClientRequestContext ctx;
-        private final RequestHeadersBuilder headersBuilder;
+        @Nullable private final RequestHeadersBuilder headersBuilder;
```

Comment:
I think we can revert the headers related changes now that we just serve the same request object

Reply:
agreed